vita3k/modules/SceDisplay/SceDisplay.cpp

```diff
@@ -52,9 +52,9 @@ static int display_wait(EmuEnvState &emuenv, SceUID thread_id, int vcount, const
     return SCE_DISPLAY_ERROR_OK;
 }
 
-EXPORT(SceInt32, _sceDisplayGetFrameBuf, SceDisplayFrameBuf *pFrameBuf, SceDisplaySetBufSync sync) {
+EXPORT(SceInt32, _sceDisplayGetFrameBuf, SceDisplayFrameBuf *pFrameBuf, SceDisplaySetBufSync sync, uint32_t *pFrameBuf_size) {
     TRACY_FUNC(_sceDisplayGetFrameBuf, pFrameBuf, sync);
-    if (pFrameBuf->size != sizeof(SceDisplayFrameBuf))
+    if (pFrameBuf->size != sizeof(SceDisplayFrameBuf) && pFrameBuf->size != sizeof(SceDisplayFrameBuf) - 4)
```

Comment:
Doesn't this need a comment? I don't think someone just looking at the code would understand why it compares pBufferBuf->size to two different sizes. This is a kind of magic that requires some background knowledge.

Reply:
You are right, I rewrote the code to make it much more readable